packages/cli/configuration/src/generators-yml/schemas/APIConfigurationV2Schema.ts

```diff
@@ -0,0 +1,26 @@
+import { z } from "zod";
+import { RawSchemas } from "@fern-api/fern-definition-schema";
+
+export const OpenAPISpecSchema = z.strictObject({
+    openapi: z.string(),
+    overrides: z.string().optional(),
+    namespace: z.string().optional()
+});
+
+export type OpenAPISpecSchema = z.infer<typeof OpenAPISpecSchema>;
+
+export const AsyncAPISchema = z.strictObject({
+    openapi: z.string(),
```

Comment:
if not loose `api` should this at least be `asyncapi` so there's distinction between the types?

Reply:
great catch, will update